docs/book/src/clusterctl/developers.md

```diff
@@ -36,15 +36,13 @@ If you want to create a local artifact, follow these instructions:
 In order to build artifacts for the CAPI core provider, the kubeadm bootstrap provider and the kubeadm control plane provider:
 
 ```
-make docker-build REGISTRY=gcr.io/k8s-staging-cluster-api
-make generate-manifests REGISTRY=gcr.io/k8s-staging-cluster-api PULL_POLICY=IfNotPresent
+make docker-build REGISTRY=gcr.io/k8s-staging-cluster-api PULL_POLICY=IfNotPresent
```

Comment:
I assume the make targets have been refactored at some point. The `docker-build` target updates the config folder with the image and the image pull policy. `generate-manifests` doesn't change anything (anymore).